lib/options.js

```diff
@@ -393,13 +400,33 @@ function refreshFilterPage() {
     // todo: gross hack, use templating framework
     var action = optionsGetAction(origin, originList);
     if(!action){ continue; }
+      ++cnt;
```

Comment:
It looks like you are increasing count a second time here, since this also hapens in addOriginHTML. Is there a reason for this?